vispy/gloo/glir.py

```diff
@@ -259,19 +260,25 @@ def as_es2_command(command):
 class BaseGlirParser(object):
     """ Base clas for GLIR parsers that can be attached to a GLIR queue.
     """
-    
+
+    def __init__(self):
+        self.capabilities = dict(
```

Comment:
It seemed to complicate the code to have to add it to `gloo/context.py` (there was no `gl` use there and there is in the `parser`, for instance), so I made a wrapper in `GLContext` to its parser. This means the API allows access as `canvas.context.capabilities['max_texture_size']`, so we can refactor later if need be. It at least gets us less painfully nested access to the variable.